FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataWriter/Azure/AzureBlobContainerClient.cs

```diff
@@ -39,19 +39,21 @@ public class AzureBlobContainerClient : IAzureBlobContainerClient
 
         public AzureBlobContainerClient(
             Uri storageUri,
+            ITokenCredentialProvider credentialProvider,
             ILogger<AzureBlobContainerClient> logger)
         {
             EnsureArg.IsNotNull(storageUri, nameof(storageUri));
+            EnsureArg.IsNotNull(credentialProvider, nameof(credentialProvider));
             EnsureArg.IsNotNull(logger, nameof(logger));
 
             _logger = logger;
 
             _blobContainerClient = new BlobContainerClient(
                 storageUri,
```

Comment:
Wha't the performance of `GetCredential`. Is it better to `GetCredential` at first, and use it in these two places?

Reply:
Thanks, fixed it.